### ISSUE TITLE
Create a Glossary

### ISSUE THREAD

**Issue by jldodds** [cjerdonek/formal-rcv#5]
There are several imprecise terms (e.g. ballot). As we come up with names for these in our specifications we should create a glossary defining them in english

**Comment by cjerdonek** [comment]
I stubbed out a glossary [here](https://github.com/cjerdonek/formal-rcv/blob/master/docs/glossary.md) and will start fleshing out more definitions.  Feel free to add to this issue any additional terms that you think should be defined in the glossary.

One thing I mentioned in the glossary intro:

I think we should consider having a standard way of visually distinguishing between terms as they are used in a statute we are discussing with terms when we are using them in the sense of our standard glossary definition.  For example, maybe we can have a convention of using all caps when using a term in the canonical project sense of the word.  Or perhaps we can prefix statute-specific references (e.g. "SF ballot").

I don't have an answer to suggest yet, but I wanted to flag this as something to consider since otherwise it may get confusing to define words used in a jurisdiction's statutes in terms of the terms as we've defined them in the glossary.  For example, for San Francisco we would want to say in a convenient way that San Francisco's "ballot" maps to the glossary's "vote".

**Comment by jldodds** [comment]
I think we should always list our word first, weather we lifted it from one of the documents or decided on it ourselves, then as part of that definition we can give the equivalent word in the other documents.

**Comment by cjerdonek** [comment]
Okay, I finished fleshing out the initial glossary.  Let me know if you think any terms should be added or if you have any suggestions or questions about how I defined any of the terms.  A lot of this is from experience.

Some comments on it:
- I noted that these definitions are _informative_ rather than _normative_.  In other words, they shouldn't be viewed as the root, "ultimate truth" definitions (those should be done in Coq).  However, the Coq definitions should be consistent with the glossary definitions, and the glossary definitions descriptive of what is written in Coq.  My view was that the glossary can be an easy, human-readable reference sheet of how we are using terms throughout the project (as well as a guide when writing the Coq).
- In terms of the Coq formalization, this will mean that we want to be using "vote" in our code rather than "ballot" when counting a voter's selections, etc.
- The phrase "physical vote" is something I made up.  Let me know if you have a better idea.  I don't think it's something we'll need to use very often (though possibly in the context of auditing).  So I added it more for the sake of completeness.  Also, since the notion of an interpreted vote rather than the raw, physical vote is the thing we'll be using for the most part throughout our project (e.g. in the code), I decided to give that the simpler name (i.e. "physical vote" interprets to "vote" rather than, say, "vote" interprets to "interpreted vote").

@jldodds We could try out the method of distinguishing the "project" vs. "statute" meaning of terms in the SF spec that @robdockins started, since that is the only place right now where statute-specific terminology appears.
